src/telliot_feeds/reporters/tips/listener/tip_listener.py

```diff
@@ -0,0 +1,57 @@
+from abc import ABC
+from abc import abstractmethod
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+from telliot_feeds.feeds import DataFeed
+
+
+class TipListener(ABC):
```

Comment:
You only want to make an abstract base class if you think us/others are going to implement different kinds of tip listeners in the future. Think so?

Also, for each of the abstract methods below, if they're not implemented in subclasses of TipListener, a not implemented error will be thrown. Do you think future versions/implementations will need each of these methods you've defined?